node/src/main/resources/defaults.conf

```diff
@@ -0,0 +1,366 @@
+# Start a stand-alone node.
+# In this mode node
+# 1. does not connect to any network
+# 2. `casper.genesis-ceremony.ceremony-master-mode` = true
+# 3. `required-signatures` = 0
+# for node to be able to create and approve genesis block if its not available
+standalone = false
+
+protocol-server {
+  # ID of the RChain network.
+  # Node shall only accept messages with this `network-id`.
+  # This is not Casper construct so it is not in `casper` section.
+  # This is used to filter out messages originated from different networks to lower down the load.
+  network-id = "testnet"
+
+  # Host name or IP address of this node. Will be used by peers to reach this node, so by default have to be public
+  # address.
+  # If this attribute is not set then node will try to guess its public IP address.
+  # Note: actual protocol server is binded to "0.0.0.0"
+  # host = localhost
+
+  # Allow `host` to be non publicly accessible address. Required for private networks installations.
+  allow-private-addresses = false
+
+  # Use random ports in case RChain Protocol port and/or Kademlia port are not free.
+  use-random-ports = false
+
+  # Support dynamic IP
+  dynamic-ip = false
+
+  # Disable UPnP
+  no-upnp = false
+
+  # Port for protocol server
+  port = 40400
+
+  # Maximum message size that can be received
+  grpc-max-receive-message-length = 256K
+
+  # Maximum size of messages that can be received via streams
+  # This is effectively max size of a block
+  grpc-max-receive-stream-message-length = 256M
+
+  # Number of incoming RChain Protocol message consumers
+  # This should be high enough, as BlockMessage consumption can hang waiting for block processing lock and
+  # occupy one consumer.
+  # The very minimum should be {number of nodes} * {synchrony constraint}, which is enough to consume
+  # blocks proposed by peers simultaneously. Taking into account blocks are not the only protocol message,
+  # and blocks can be received as well via `requesting missing blocks` routine, it is recommended to
+  # make this number high. Upper bounded by the numer of messages (blocks are the biggest of them)
+  # node is able to keep in memory. Block size upper bound size is `max-stream-message-size`.
+  max-message-consumers = 400
+}
+
+protocol-client {
+  # Should be the same as server network ID.
+  network-id = ${protocol-server.network-id}
```

Comment:
Since the client `network-id` and the server `network-id` should be the same. I think it is better to only config one param.

Reply:
I wanted to as much as possible show all info that is required for different parts of the system. Protocol client requires to be initiated with right network-id for its messages to be accepted by peers. Might be its too hard wording `Should be the same as server network ID`, its just an expected behaviour. I'd like to keep this duplicate, as a part of attempt to make node modular.